mkt/developers/models.py

```diff
@@ -193,41 +191,6 @@ class Meta:
         db_table = 'user_inapp_keys'
 
 
-class PreloadTestPlan(ModelBase):
-    addon = models.ForeignKey('webapps.Webapp')
```

Comment:
hm okay, i think i can just remove the foreignkey field and keep the table around (for now)